src/modules/languages/rust.nix

```diff
@@ -63,6 +63,19 @@ in
 
   config = lib.mkMerge [
     (lib.mkIf cfg.enable {
+
+      # Set $CARGO_INSTALL_ROOT so that executables installed by `cargo install` can be found from $PATH
+      enterShell = ''
+        export CARGO_INSTALL_ROOT=$(${
+          lib.strings.escapeShellArgs [
+            "${pkgs.coreutils}/bin/realpath"
+            "--no-symlinks"
+            "${config.devenv.state}/cargo-install"
```

Comment:
Where will this install the binaries? 

I'm a bit confused, doesn't `$CARGO_INSTALL_ROOT` tell where the binaries should be installed for when you call cargo-install?

It would also be great if this added a test into `.test.sh` to assure we can run the binary.

Reply:
Yes, `$CARGO_INSTALL_ROOT` tell where the binaries should be installed for when you call `cargo install`. What confuses you?

Reply:
@domenkozar Tests added